### PR TITLE
RPED drops lowest tier parts

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
see title
## Why It's Good For The Game
i like being able to drop the shittiest parts i have first
## Changelog
:cl:
tweak: RPEDs now drop their lowest part tier first when quick-emptied (used inhand).
/:cl:

### CODE REVIEW

File: code/datums/components/storage/concrete/rped.dm [hunk 1]
```diff
@@ -33,5 +59,29 @@
 			to_chat(M, "<span class='warning'>[parent] only accepts machine parts!</span>")
 		return FALSE
 
-/datum/component/storage/concrete/cyborg/rped
```
Comment (SandPoot) [inline code/datums/components/storage/concrete/rped.dm:36]: accidental delet?

Comment (Hatterhat) [reply code/datums/components/storage/concrete/rped.dm:36]: intentional, it's unused

File: code/datums/components/storage/concrete/rped.dm [hunk 2]
```diff
@@ -16,6 +16,32 @@
...
+	A.add_fingerprint(M)
+	var/list/things = contents()
+	var/lowest_rating = INFINITY
+	for(var/obj/item/B in things)
+		if(B.get_part_rating() < lowest_rating)
+			lowest_rating = B.get_part_rating()
+	for(var/obj/item/B in things)
+		if(B.get_part_rating() > lowest_rating)
+			things.Remove(B)
+	if(lowest_rating = INFINITY)
```
Comment (DeltaFire15) [inline code/datums/components/storage/concrete/rped.dm]:
```suggestion
	if(lowest_rating == INFINITY)
```

File: code/datums/components/storage/concrete/rped.dm [hunk 3]
```diff
@@ -33,5 +59,29 @@
...
```
Comment (DeltaFire15) [inline code/datums/components/storage/concrete/rped.dm]:
```suggestion
	if(lowest_rating == INFINITY)
```

**Review by DeltaFire15 — COMMENTED** [review]
Fix these to get travis to stop screaming.

Also localtest it for safety I guess.

**Review by DeltaFire15 — APPROVED** [review]
Seems fine and I didn't see anyone complain while it was testmerged.

**Pull request merged**